control-plane-operator/controllers/hostedcontrolplane/csi/kubevirt/kubevirt.go

```diff
@@ -396,6 +436,30 @@ func ReconcileTenant(client crclient.Client, hcp *hyperv1.HostedControlPlane, ct
 		return err
 	}
 
+	externalSnapshotControllerServiceAccount := manifests.KubevirtCSIDriverExternalSnapshotSA(tenantNamespace)
+	_, err = createOrUpdate(ctx, client, externalSnapshotControllerServiceAccount, func() error {
```

Comment:
we shouldn't need a new SA. also it doesn't appear that this SA is attached to any pods?